api/v3/MailingEventSubscribe.php

```diff
@@ -46,6 +46,8 @@ function civicrm_api3_mailing_event_subscribe_create($params) {
   $group->is_active = 1;
   $group->id        = (int) $group_id;
   if (!$group->find(TRUE)) {
+    fprintf(STDERR, "Is this it\n");
+    throw new \Exception('Maybe this is it');
```

Comment:
hmm hopefully it wouldn't even get that far - lessee

Reply:
ok I guess I still don't understand how those tests work. I think you're saying somewhere before this the api system validates if group_id is an integer to begin with, and there's a test somewhere that tests that.

Reply:
@demeritcowboy yeah - well I guess I didn't double check the metadata - but if the metadata declares the field as a pseudoconstant then the api wrapper validates that value agains the pseudoconstant options (both keys & names)

Reply:
@demeritcowboy I'm thinking it might have something to do with this https://github.com/civicrm/civicrm-core/pull/26931/files#diff-4b5d2fddbdfa4e503cc1050d03e4ab9c95ffd52870133980c2df6159007a1637R92

Reply:
Ok thanks. I'm still not sure which test tests invalid data. Will see if I can set a trap for it.